specs/player_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+  describe "#initialize" do
+    it "Takes a name" do
+      player = Scrabble::Player.new("Erika")
+
+      player.must_respond_to :name
+      player.name.must_equal "Erika"
+      player.name.must_be_kind_of String
+    end
+  end
+
+  describe 'plays' do
+    it 'Returns an array of played words' do
+      word = "dikla"
+      user = Scrabble::Player.new(word)
+      scrabble_word = user.plays
+      scrabble_word.must_be_kind_of Array
+    end
+  end
+
+  describe 'play' do
+    it 'Adds the input word to the plays Array' do
+
```

Comment:
What happens if the player has already won? Is the word recorded? What does it return?